.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5503
+}
```

Comment:
you can hide this file with gitignore. 
I don't know how to do that but Vikram once suggested doing that

Reply:
Yes, I will try that.